source/Playnite/GamesEditor.cs

```diff
@@ -850,15 +851,15 @@ public void RemoveGame(Game game)
                     new MessageBoxOption("LOCNoLabel", false, true)
                 };
                 var result = Dialogs.ShowMessage(
-                    "LOCGameRemoveAskMessageIgnoreOption",
+                    string.Format(resources.GetString("LOCGameRemoveAskMessageIgnoreOption"), game.Name),
                     "LOCGameRemoveAskTitle",
                     MessageBoxImage.Question,
                     options);
                 if (result == options[0])
                 {
                     addToExclusionList = true;
                 }
-                else if (result == options[2])
+                else if (result == null || result == options[2])

```

Comment:
FYI this is something I noticed where if you were to close the game via something other than the action buttons on the dialog (e.g alt-tab and close the dialog), then the logic would proceed to remove the game even though this should more naturally cancel the operation (i.e not remove the game).